examples/tokenvm/actions/create_asset.go

```diff
@@ -21,6 +21,9 @@ type CreateAsset struct {
 	// Metadata is creator-specified information about the asset. This can be
 	// modified using the [ModifyAsset] action.
 	Metadata []byte `json:"metadata"`
+
+	// Adding a hardcap. If 0, no hardcap exists (and will be MaxUint)
+	MaxSupply uint64 `json:"maxSupply"`
```

Comment:
I don't think we should allow `0` here. If you are creating an asset, then your MaxSupply will be greater than 0.

Reply:
`0` was meant as default-value and thus MaxUint64 like it has been before this change. Shall we we force to make a value mandatory? What do you think @patrick-ogrady?